cmake/libprotobuf.cmake

```diff
@@ -26,8 +26,8 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
   target_link_libraries(libprotobuf PRIVATE log)
 endif()
 target_include_directories(libprotobuf PUBLIC ${protobuf_SOURCE_DIR}/src)
-target_link_libraries(libprotobuf PRIVATE ${protobuf_ABSL_USED_TARGETS})
-target_include_directories(libprotobuf PRIVATE ${ABSL_ROOT_DIR})
+target_link_libraries(libprotobuf PUBLIC ${protobuf_ABSL_USED_TARGETS})
```

Comment:
Can you update the other cmake files while you're at it?  e.g. libprotoc, protoc, libprotobuf-lite, conformance

Reply:
Done :)